tests/build.proj

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="dir.props" />
+  <Import Project="Directory.Build.props" />
```

Comment:
I think the correct solution for build.proj is to convert it to an sdk style project.

Reply:
Converting .proj files into sdk style projects does odd things. If we need to do that, it should be a separate change.